addons/maintenance/models/maintenance.py

```diff
@@ -154,6 +155,19 @@ def _compute_display_name(self):
     color = fields.Integer('Color Index')
     scrap_date = fields.Date('Scrap Date')
     maintenance_ids = fields.One2many('maintenance.request', 'equipment_id')
+    equipment_properties = fields.Properties('Properties', definition='category_id.equipment_properties_definition', copy=True)
+    match_serial = fields.Boolean(compute='_compute_match_serial')
+
+    @api.depends('serial_no')
+    def _compute_match_serial(self):
+        matched_serial_data = self.env['stock.lot']._read_group(
+            [('name', '=', self.serial_no)],
```

Comment:
This compute doesn't work with recordset (traceback if you want to display it in the list view for example)
You can make it multi
```suggestion
            [('name', 'in', self.mapped('serial_no'))],
```